* TODO Items
** Phylogenetic Survey Paper     
*** Specific TODO items
**** TODO Fix Table 1
     * Make table 1 more readable with better labels etc.
**** TODO Quality information
     * Display quality information in the supplemental material
     * CheckM values, coverage, quality scores
     * Cite these qualities where relevant in the results
**** TODO Visualization with plasmid distribution
**** TODO Visualization with taxonomic distribution
**** TODO Explain green dots
     * eplain the green dots
     * look for other missing information in the figures
*** General TODO items
**** TODO Gradient of the AMR trees
     * Come up with a better term/description of the observed “gradient of the AMR trees”, where the branches gradually could become less likely to be AMR determining the further away we get from canonical clades.
     * We have all these clades of diversity peppered throughout.
     * This should probably be briefly described in the abstract and results.
     * Maybe it already is summed up by diversity/spread type talk, and this will just sound repetitive.
**** TODO Results 
     * We still do not have a clear picture.
     * Yes, we can see that some of these things are probably poorly named when it comes to their phylogenetic relationships. (Could there be an analogy to multi locus sequence typing here?)
     * Yes, we can see that there may be some novel AMR genes (in the MCR trees)
     * Yes, we can see that there may be some problems making other trees because of plasmids etc.
     * But, can we say anything solid about this?
**** TODO Conclusion 
     * If the results were more focused, a conclusion would come naturally.
     * This conclusion will need to be added to Abstract and Conclusion section.
**** TODO Consider rebuilding some of the trees
     * Example: Acinetobacter is not showing up in the MCR trees, and it had been discussed earlier that it may be taxonomically interesting in it's placement in the tree. (This was in notes from a meeting this summer, but it would require looking at the raw tree to see where it is located)
     * There could be other relationships we are missing out on because of the clustering.
     * Decide how much we want to discuss these sorts of interesting relationships, or if it's worth it to rebuild the trees based on more "interesting" taxa.
**** TODO Discussion section
     * We already have many discussion points, and it should be easy to write a lot of this section independently of the other TODO items...
     * Messed up nomenclature
     * We found a lot of intermingled MCRs, not so much with the others (plasmids etc.)
     * Why were some UBAs not covered? (plasmids)
     * What should we do with naming, especially as new variants are discovered and fill in the tree?
     * Taxonomic anomalies and how they may be explained
     * There were many clinical samples represented that did yield MCRs, but the recovery could be bad (This was another item from meeting notes of which I am not sure of the meaning)
