## Abstract {.page_break_before}

Antimicrobial resistance (AMR) is a global threat to human health.
Efforts to prevent the spread of AMR rely on surveillance of possible AMR determinants.
Using phylogenetics to highlight potential sources of AMR could guide researchers in chosing organisms for phenotypic resistance testing. 
A systematic way of classifying AMR gene variants is an important in comparing these phylogenetic relationshits, facilitating research in the feild.
This study examines relationshipts within sets of previously classified canonical AMR family variants to demonstrate their usefulness in identifying AMR candidates.
Canonical sequence data from the Comprehensive Antibiotic Resistance Database (CARD) was used to query CARD prevalence data, NCBI sequence data, and predicted resistomes in in a data set of 7903 draft quality metagenome assembled genomes @doi:10/cczd  (MAGs) from various uncultivated bacterial and archeal sources (UBAs).
Genes associated potentially associated with mobile colistin resistance (MCR) were found in the UBA sources.
New Dheli beta-lactamases (NDM), Klebsiella pneumoniae carbapenemases (KPC), and OXA beta-lactamases were not found to be represented in the UBAs, but
[TODO: "gradient of the AMR trees"]
[TODO: conclusion]

