## Abstract {.page_break_before}

Antimicrobial resistance (AMR) is a global threat to human health.
Efforts to prevent the spread of AMR rely on surveillance of possible AMR determinants.
Using phylogenetics to highlight potential sources of AMR could guide researchers in choosing organisms for phenotypic resistance testing.
A systematic way of classifying AMR gene variants is important in comparing these phylogenetic relationships.
In this study, the phylogenetic neighborhoods of several named AMR genes are examined for diversity, spread, and consistency in classification.
Canonical sequence data from the Comprehensive Antibiotic Resistance Database (CARD) for mobile colistin resistance (MCR), New Delhi beta-lactamase (NDM), Klebsiella pneumoniae carbapenemase (KPC), and OXA beta-lactamase were used to query CARD prevalence data, NCBI sequence data, and draft quality metagenome assembled genomes (MAGs) from various uncultivated bacterial and archeal sources (UBAs).
A phylogeny of this expanded set of candidate resistance genes was produced with a limited amount of new diversity, most notably with genes which were potentially associated with MCR.
In this phylogenetic context, some of the names of the genes were found to be inconsistent or inconsequential.
