## Abstract {.page_break_before}

Antimicrobial resistance (AMR) is a global threat to human health.
Efforts to prevent the spread of AMR rely on surveillance of possible AMR determinants.
Using phylogenetics to highlight potential sources of AMR could guide researchers in choosing organisms for phenotypic resistance testing. 
A systematic way of classifying AMR gene variants is important in comparing these phylogenetic relationships.
In this study, the phylogenetic neighborhoods of several named AMR genes are characterized by their diversity, spread, and potential for discovering possibly novel AMR variants.
Canonical sequence data from the Comprehensive Antibiotic Resistance Database (CARD) was used to query CARD prevalence data, NCBI sequence data, and draft quality metagenome assembled genomes (MAGs) from various uncultivated bacterial and archeal sources (UBAs).
Genes which were potentially associated with mobile colistin resistance (MCR) were found in the UBA sources.
New Delhi beta-lactamases (NDM), *Klebsiella pneumoniae* carbapenemases (KPC), and OXA beta-lactamases were not found to be represented in the UBAs,
[TODO: "gradient of the AMR trees"]
[TODO: conclusion: lots of novelty that cannot be accommodated by existing nomenclature schemes?]
