## Abstract {.page_break_before}

Antimicrobial resistance (AMR) is a global threat to human health.
Efforts to prevent the spread of AMR rely on surveillance of possible AMR determinants.
Under-sampled, and uncultured microbial genomes in the form of metagenome data, are a potential source for AMR gene discovery.
The Resistance Gene Identifier (RGI) and the Comprehensive Antibiotic Resistance Database (CARD) @doi:10/f9wbjs were used to predict resistomes in a data set of 7903 draft quality metagenome assembled genomes @doi:10/cczd  (MAGs) from various uncultivated bacterial and archeal sources (UBAs).
Genes associated with mobile colistin resistance (MCR), and carbapenem resistance, such as Metallo-beta-lactamases (NDM), Klebsiella pneumoniae carbapenemases (KPC), and OXA beta-lactamases were found to be represented in the UBAs.
[TODO: findings]
[TODO: conclusion]

