## Abstract {.page_break_before}

Antimicrobial resistance (AMR) is a global threat to human health.
Efforts to prevent the spread of AMR rely on surveillance of possible AMR determinants.
Phylogenetic investigations of the evolution and transmission of antimicrobial resistance determinants can be used to guide and prioritize experimental molecular studies.
A systematic way of classifying AMR gene variants is important in comparing these phylogenetic relationships.
In this study, the phylogenetic neighborhoods of several named AMR genes are examined for diversity, spread, and consistency in classification.
Canonical genes which are associated with mobilizable, last-line antibiotics were chosen from the Comprehensive Antibiotic Resistance Database (CARD) to query CARD prevalence data, NCBI sequence data, and draft quality metagenome assembled genomes (MAGs) from various uncultivated bacterial and archeal sources (UBAs).
These genes included mobile colistin resistance (MCR), New Delhi beta-lactamase (NDM), Klebsiella pneumoniae carbapenemase (KPC), and OXA-48 beta-lactamase lactamase family.
A phylogeny of this expanded set of candidate resistance genes was produced with a limited amount of new diversity in the beta-lactamase analyses, and several possibly novel clades of diversity in MCR.
The beta-lactamase families were found to be more tightly defined, with most added diversity being either extremely close, or phylogenetically distant.
The MCR family showed many new clades of diversity within canonical clades.
In this phylogenetic context, some of the names of the genes were found to be inconsistent or inconsequential.
