## Introduction
An important part of the microbial adaptive process is an organisms ability to adapt to pressure from antimicrobial agents in the environment. This allows microbes to continue to grow despite being in the presence of a compound designed to inhibit the growth or kill them @https://www.cdc.gov/drugresistance/about.html. This is known as antimicrobial resistance (AMR). AMR within pathogenic microbes is a threat to our treatment of infectious diseases, with many antibiotics becoming less effective @https://amr-review.org/sites/default/files/160525_Final%20paper_with%20cover.pdf. AMR is a growing global health crisis with the spread of AMR leading to prolonged and more severe infections, decreased protection after surgery, and increased morbidity @doi:gfnrkj. It has been estimated that by 2050 AMR will be responsible for 10 million deaths per year @https://amr-review.org/sites/default/files/160525_Final%20paper_with%20cover.pdf. 

AMR is a natural survival mechanism that has been around long before humansâ€™ widespread use of antibiotics @doi:b3wbvx. A potential source of discovery of new AMR determinants is an examination of Under-sampled and uncultured microbial genomes in the form of metagenome data. The Uncultivated Bacteria and Archaea (UBA) dataset consists of 7903 metagenomes recovered from 1500 public metagenomes from environmental and animal gut samples @doi:cczd. 

AMR is becoming more prevalent, and is even starting to become an issue with last-line antibiotics such as colistins, and carbapenemases @https://amr-review.org/sites/default/files/160525_Final%20paper_with%20cover.pdf. 
This problem is arising due to irresponsible practices in healthcare, and in agriculture. Much of this problem is due to indiscriminate application of antibiotics, without regard for current knowledge on the matter. Another portion of this problem is due to a lack of insight into the constantly changing pathogenic resistomes, where bad practices could be carried out in ignorance. In an effort to control the spread of AMR, it is important that we collect as much information as possible about this process. Surveillance and characterization of many AMR genes is currently done through curated databases such as the Comprehensive Antibiotic Resistance Database (CARD). These databases contain plenty of AMR information about resistomes which are commonly studied, but it is possible that a lack of important information exists in the rest of the environment. Observing potential AMR genes for antibiotics such as carbapenemases and colistins, outside of a clinical setting, could lead to valuable new information.
A potential source of discovery of new AMR determinants is an examination of Under-sampled, and uncultured microbial genomes in the form of metagenome data. The Resistance Gene Identifier (RGI) uses the Comprehensive Antibiotic Resistance Database (CARD) @doi:10/f9wbjs to predict potential AMR genes from sequence data, would be a useful tool in examining such data. Using canonical reference genes associated with colistin and carbapenem resistance, we can use current knowledge to query these metagenomes, an perhaps glean new knowledge from the phylogenetic relationships.
Carbapenems and polymyxins are last line antibiotics that are used to treat patients infected with bacteria that are resistant to other antibiotics. However, there has been a recent emergence of resistance to these last line antimicrobials, presenting a new threat to public health @doi:10.1111/1469-0691.12719
TODO: introduce MCR, OXA @doi:10.1128/CMR.00117-13, KPC, and NDM
