## Introduction

Antimicrobials, substances which can kill or inhibit the growth of microbes [@doi:10/dbhq], are key effectors in the ecology of microorganisms.
The evolution of resistance to these substances, known as antimicrobial resistance (AMR), is an important and ancient adaptive process [@doi:10/b3wbvx].
In the last century we have adopted and used antimicrobials to great effect in clinical medicine and agriculture.
However, the use and abuse of antimicrobials has led to increasing levels of observed AMR [@doi:10/f9wbjs].
This poses a growing global health risk by undermining our ability to treat infectious diseases and perform surgeries [@doi:10/gfnrkj].
Infections by resistant microbes have greater mortality and morbidity [@doi:10/gfnrkj] with the European Centre for Disease Prevention and Controlling estimating that 25,000 people per year die due to AMR [@doi:10/c9zd].

AMR can be due to intrinsic resistance within the organism or the acquisition of resistance via lateral gene transfer [@doi:10/b3wbvx].
These mechanisms allow for the constant change of the resistome, making determining the scope of AMR within both pathogenic and non-pathogenic bacteria difficult [@doi:10/f9wbjs].
To mitigate the spread of AMR the World Health Organization (WHO) created an action plan which emphasizes the need to strengthen our understanding and surveillance of AMR. 
An important aspect of this research is to examine the spread of AMR between environmental and clinical samples (https://www.who.int/antimicrobial-resistance/global-action-plan/en/).
One way to try to understand the evolution and spread of AMR is to perform large-scale evolutionary analyses with samples from lots of different environments/contexts.

Some of the most critical pathogens are the multi-drug resistant "ESKAPE" pathogens: _Enterococcus faecium_, _Staphylococcus aureus_, _Klebsiella pneumoniae_, _Acinetobacter baumannii_, _Pseudomonas aeruginosa_, and _Enterobacter_ spp. [@doi:10/f9h4q3].
These are the leading cause of hospital-acquired infections and are largely only treatable with carbapenem or polymyxin antibiotics [@doi:10/c9zd].
Carbapenems are broad spectrum beta-lactam antibiotics typically used to treat life threatening, high risk, and multi-drug resistant bacterial infections. 
beta-lactamase producing bacteria have difficulty degrading carbapenems when combined with beta-lactamase inhibitors making them effective treatments against pathogens with multi-drug resistance [@doi:10/c9zd].
However, there has been a recent emergence of resistance to these antimicrobials, presenting a new threat to public health. 
One way microbes can develop carbapenem resistance is through the acquisition of carbapenamase genes. [@doi:10/c9zd].
Three main carbapenamase genes of concern are _Klebsiella pneumoniae_ carbapenamase (KPC), New Delhi metallo-beta-lactamase (NDM) and OXA beta-lactamases [@doi:10/c9zd].

Carbapenamases are classified as one of four classes (A-D) based on their molecular structure [@doi:10/cdckks].
Class A, C, and D both use a serine residue to hydrolyze beta-lactam antibiotics [@doi:10/bp6dp2].
KPC genes are a part of class A. 
Many variants of KPC are spreading and being discovered, however carbapenamases within this class are more rare than the other three classes [@doi:10/c9zd].
Class D contains the OXA family of beta-lactamases. 
Unlike KPC and NDM, this family is characterized by phenotype rather than genotype. 
This family is characterized by specific hydrolases that can hydrolyze oxacillin and cloxacillin, with the enzyme being poorly inhibited by clavulanic acid (https://card.mcmaster.ca/ontology/36026).
This results in a low amount of sequence homology within the family. 
A subset of this family, OXA-48 possesses carbapenem hydrolyzing activity [@doi:10/cn235v].
The OXA-48 subfamily consists of five variants: OXA-48, OXA-162, OXA-163, OXA-181, OXA-204, and OXA-232 [@doi:10/dbhs].
Class B beta-lactamases are metallo-enzymes that use a zinc active site to hydrolyze beta-lactam antibiotics. 
The NDM gene is classified as a class B beta-lactamase [@doi:10/c9zd].
When pathogens become resistant to carbapenems, the last line of defense is the polymyxin antibiotic colistin. 
Polymyxins act via disrupting membrane permeability @doi:10/dbht. 
Colistin is limited to a last line defense against carbapenem resistant infections due to its neurotoxic and nephrotoxic effects [@doi:10/dbht]. 
Despite its already limited use, mobilized colistin resistance (MCR) have been discovered [@doi:10/gdqcfq]. 
Due to the emergence of resistance to many last-line antibiotics, it is important to characterize the total diversity and spread of these key resistance determinants.
However, to perform these types of analyses we need two resources: a clear consolidated definition of what constitutes an AMR-related gene and a large amount of sequencing microbial genomes.

Naming and defining AMR genes is difficult due to the legacy of research pre-DNA sequencing technology and the large number of different researchers and stakeholders involved in AMR.
AMR determinants are typically classified into AMR families, some of which are based on phenotypic properties, and some based on sequence variation.
The nomenclature is usually an acronym representing the mechanism of resistance, along with a numerical value to distinguish variants. 
Each sequence, determined to be novel by some arbitrary criteria, is assigned a new number.
Sequences are further sub-categorized when the sequence similarity is high, and as much as a single amino acid difference has given rise to a newly named determinant.
This system for classification has the potential to be misleading when conducting AMR research.
AMR families could appear to have a large amount of diversity, when in reality, sequences are closely related, and only a small number are actually distinct.
Additionally, sequences which are not homologues, could potentially be classified in the same family, simply based on their function.
This is a problem when attempting to characterize the AMR determinants by sequence similarity.
[TODO: analogy to multi-locus sequence typing?]

High-quality manually curated AMR databases such as the Comprehensive Antibiotic Resistance Database (CARD) [@doi:10.1093/nar/gkz935] provide a unified resource for the definition, nomenclature and classification of AMR genes.
CARD organises this information CARD through the antibiotic resistance ontology (ARO), a controlled vocabulary with defined relationships.
There are two data-sets within CARD, canonical and prevalence.
Canonical is a conservative set of AMR genes and mutations that have been experimentally verified as being associated with resistance in a peer-reviewed publication.
This is therefore slightly biased towards the organisms that have been most heavily studied.
On the other hand, prevalence contains AMR sequences that have been detected by searching a broader set of WHO priority organism genomes from repositories such as National Center for Biotechnology Information (NCBI) for sequences similar to canonical sequences.
This _in silico_ search is performed using CARD's BLAST-based Resistance Gene Identifier (RGI) tool and greatly increases the sequence diversity in CARD [@doi:10.1093/nar/gkz935].

However, this is still limited to a subset of all genomes currently available in central repositories like NCBI.
If we want to thoroughly understand the evolution and spread of AMR genes we need to analyse as many genomes as possible.
Unfortunately, the genomes in databases are largely sequenced from microbes that can be easily cultured. 
As only a subset of microbial diversity can be cultured, this means many of our existing genomes aren't necessarily representative of the environment from which they were sampled.
Recently, techniques have been developed that allow the recovery of genomes from metagenomic data e.g. [@doi:10.7717/peerj.1165,@doi:10.1038/s41564-018-0171-1].
As metagenomic sequencing, the direct sequencing of all DNA in a sample, doesn't require culturing these metagenome-assembled genomes (MAGs) represent a huge source of novel microbial diversity.
Parks et. al. [@doi:10.1038/s41564-017-0083-5], generated new 7,903 bacterial and archael (Uncultured Bacteria and Archaea; UBA) MAGs representing >30% increase in the sampled phylogenetic diversity of the bacteria and archaea.

Identifying and phylogenetically analysing key carbapenemase and colistin resistance genes in this dataset, CARD, CARD-prevalence and NCBI genomes, could greatly improve AMR surveillance of these genes.
We would characterise previously unseen diversity in genomes not yet analyzed for AMR, and provide insights into the diversity of AMR across non-clinical samples.
This could inform our understanding of the transmission of these mobile critical AMR genes and help refine their current nomenclature.
Therefore, in this work we present a comprehensive phylogenetic survey and analysis of KPC, NDM, OXA-48, and MCR across all currently sequenced genomes and large sets of previously uncharacterised metagenome-assembled genomes.
