## Introduction

Antimicrobials are substances that can halt the growth of and/or kill microorganisms @doi:10/dbhq. 
An important part of the microbial adaptive process is the ability to adapt to pressure from antimicrobial agents within the environment. 
This ability allows microbes to continue to live and grow in the presence of antimicrobial compounds. 
This is known as antimicrobial resistance (AMR) https://www.cdc.gov/drugresistance/about.html. 
The abuse and misuse of antimicrobials has led to increasing levels of AMR within clinical settings @doi:10/f9wbjs.

The spread of AMR is a growing global health crisis. 
AMR within pathogenic microbes is a threat to our treatment of infectious diseases, reducing the efficacy of antibiotics, allowing for prolonged and more severe infections, and increased mortality @doi:10/gfnrkj. 
The European Centre for Disease Prevention and Control estimates that 25,000 people die per year due to infections related to AMR @doi:10/c9zd, with reports showing the annual cost to combat AMR infections in just the USA alone to be between $21 billion and $34 billion @doi:10/cgt3qk.

Microbes can be intrinsically resistant to AMR or acquire resistance via lateral gene transfer @doi:10/b3wbvx. 
These mechanisms allow for the constant change of the resistome, making determining the scope of AMR within both pathogenic and non-pathogenic bacteria difficult @doi:10/f9wbjs. 
To impede the spread of AMR the World Health Organization (WHO) created an action plan which emphasizes the need to strengthen the knowledge and surveillance of AMR. An important aspect of this research is to examine the spread of AMR between environmental and clinical samples url: https://www.who.int/antimicrobial-resistance/global-action-plan/en/. 
Characterization of many AMR genes is currently being done through curated databases. The Comprehensive Antibiotic Resistance Database (CARD) is an example of one of these curated databases. 
CARD contains reference data pertaining to the genes, proteins and mutations involved in AMR. 
CARD allows for the identification of resistance genes within a genome using the Resistance Gene Identifier (RGI). @doi:10/f9wbjs.

These curated databases contain a large amount of information on the resistomes that are most commonly studied, especially those pertaining to _Staphylococcus aureus_, _Escherichia coli_, _Klebsiella pneumoniae_, _Acinetobacter baumannii_, and _Pseudomonas aeruginosa_. 
These pathogens are multidrug resistant and are the leading cause of hospital acquired infections @doi:10/f9h4q3. 
Most antibiotics have no efficacy on these infections and are therefore treated with carbapenems or polymyxins @doi:10/c9zd.

Carbapenems are broad spectrum beta-lactam antibiotics typically used to treat life threatening, high risk, and multidrug resistant bacterial infections. 
beta-lactamase producing bacteria have difficulty degrading carbapenems when combined with beta-lactamase inhibitors making them effective treatments against pathogens with multi-drug resistance @doi:10/c9zd. 
However, there has been a recent emergence of resistance to these antimicrobials, presenting a new threat to public health. 
One way microbes can develop carbapenem resistance is through the acquisition of carbapenamase genes. @doi:10/c9zd. 
Three main carbapenamase genes of concern are Klebsiella pneumoniae carbapenemase (KPC), New Delhi metallo-beta-lactamase (NDM) and OXA beta-lactamases @doi:10/c9zd.

Carbapenamases are classified as one of four classes (A-D) based on their molecular structure @doi:10/cdckks. 
Class A, C, and D both use a serine residue to hydrolyze beta-lactam antibiotics @doi:10/bp6dp2. 
KPC genes are a part of class A. 
Many variants of KPC are spreading and being discovered, however carbapenamases within this class are more rare than the other three classes @doi:10/c9zd. 
Class D contains the OXA family of beta-lactamases. 
Unlike KPC and NDM, this family is characterized by phenotype rather than genotype. 
This family is characterized by specific hydrolases that can hydrolyze oxacillin and cloxacillin, with the enzyme being poorly inhibited by clavulanic acid https://card.mcmaster.ca/ontology/36026. 
This results in a low amount of sequence homology within the family. 
A subset of this family, OXA-48 possesses carbapenem hydrolyzing activity @doi:10/cn235v. 
The OXA-48 subfamily consists of five variants: OXA-48, OXA-162, OXA-163, OXA-181, OXA-204, and OXA-232 @doi:10/dbhs. 
Class B beta-lactamases are metalloenzymes that use a zinc active site to hydrolyze beta-lactam antibiotics. 
The NDM gene is classified as a class B beta-lactamase @doi:10/c9zd.

When pathogens become resistant to carbapenems, the last line of defense is colistin. 
Colistin is a polymyxin antibiotic. 
Polymyxins act via disrupting membrane permeability @doi:10/dbht. 
Colistin is limited to a last line defense against carbapenem resistant infections due to its neurotoxic and nephrotoxic effects @doi:10/dbht. 
Despite its already limited use, mobilized colistin resistance (MCR) have been discovered @doi:10/gdqcfq. Due to the emergence of resistance to many last-line antibiotics, it is important to characterize the total diversity and spread of these resistance determinants.

The UBA dataset contains the data needed to expand AMR surveillance to a wide range of environmental samples. 
Coupling RGI predictions of the UBA metagenomes with phylogenetic analysis of the predicted AMR genes enables the possibility to discover previously unseen AMR diversity in genomes not yet analyzed for AMR, and provides insight into the diversity of AMR across non-clinical samples.
