## Methods

### AMR Prediction

### Materials

A data-set of 7903 draft quality MAGs which were recovered from the Sequence Read Archive by Parks @doi:10/cczd were used for this analysis. These genomes were chosen specifically because they were likely to be from lineages which were under-sampled. Environmental and non-human gastrointestinal samples were the main focus of these samples.

RGI version 5.0 with CARD database version 3.02  @doi:10/f9wbjs was run on the contigs of the 7903 MAGS with the inclusion of loose, perfect, and strict hits.

CARD prevalence data version 3.0.4 ("based on sequence data acquired from NCBI on 28-Feb-2019, analyzed using RGI 4.2.2 (DIAMOND homolog detection) and CARD 3.0.1") was used to query the UBA data.
[TODO: Other versions of software]

### Obtaining reference sequences

32 canonical MCR sequence variants, 14 canonical NDM sequence variants, 6 canonical OXA-48-like sequence variants, and 18 canonical KPC sequence variants (see Table S1) as indicated by CARD for the , were obtained from the CARD database. 

In the case of OXA beta-lactamase, only OXA-48-like genes were used for analysis. The OXA family is characterized by phenotype rather than genotype, and results in a low amount of sequence homology within the family. 
The phenotype of OXA-48 results from carbapenem hydrolyzing activity @doi:10/cn235v.
This subfamily of OXA contains homologous sequences suitable for this study.
Incorperating other subfamilies of OXA proved to be too cumbersome.

Additional putative AMR gene family sequences were accumulated by querying the CARD prevalence data.
The prevalence data was translated to a BLAST database.
The CARD canonical sequences from each family were used to perform a multiple query BLASTP against the prevalence blast database with a e-value threshold and query coverages shown in Table {@tbl:exp-params} 

Many of the sequences are nearly identical, thus they were further processed by clustering with CD-HIT at a minimum sequence identities indicated also in Table {@tbl:exp-params}

The CARD canonical sequences were also used to perform a multiple query BLASTP against the NCBI non-redundant database blast database with a e-value threshold, and query coverage indicated in Table  {@tbl:exp-params}

For simplicity in identifying the taxonomic history of the non-redundant hits, MULTISPECIES sequences were removed from the analyses.
There are many highly sampled taxa and genes in the non-redundant database. To balance the distribution, and reduce the size of the non-redundant sequence set, CD-HIT was used to cluster the data as per table {@tbl:exp-params}.
 
Sequences possibly containing AMR gene prediction data for each gene family was produced by filtering RGI output based on the its association with the search strings for each determinant in Table {@tbl:exp-params}
The filtered data were translated to a blast database.
The CARD canonical sequences were used to perform a multiple query BLASTP against this UBA blast database with a e-value threshold and  query coverage in Table {@tbl:exp-params}.

<table>
<td>
||
|-|
Table: e-value, query coverage, and cluster percentage used for each AMR family experiment for the prevalence, ncbi non-redundant, and UBA databases. {#tbl:exp-params}
</td>
<tr><th>(A) MCR phosphoethanolamine transferase </th>
<th>(B) NDM beta-lactamase</th>
<th>(C) OXA beta-lactamase</th>
<th>(D) KPC beta-lactamase</th></tr>
<tr><td>


| | |
|-----------------|-------------------------------------|
| Tree            | LG+I+G4                             |
| e-value_p       | 1e-160                              |
| e-value_n       | 1e-160                              |
| e-value_u       | 1e-160                              |
| coverage_p      | 98                                  |
| coverage_n      | 98                                  |
| coverage_u      | 98                                  |
| clustering_p    | N/A                                 |
| clustering_n    | N/A                                 |
| clustering_u    | N/A                                 |

</td><td>

| | |
|-----------------|-------------------------------------|
| Tree            | LG+I+G4                             |
| e-value_p       | 1e-160                              |
| e-value_n       | 1e-160                              |
| e-value_u       | 1e-160                              |
| coverage_p      | 98                                  |
| coverage_n      | 98                                  |
| coverage_u      | 98                                  |
| clustering_p    | N/A                                 |
| clustering_n    | N/A                                 |
| clustering_u    | N/A                                 |

</td><td>

| | |
|-----------------|-------------------------------------|
| Tree            | LG+I+G4                             |
| e-value_p       | 1e-160                              |
| e-value_n       | 1e-160                              |
| e-value_u       | 1e-160                              |
| coverage_p      | 98                                  |
| coverage_n      | 98                                  |
| coverage_u      | 98                                  |
| clustering_p    | N/A                                 |
| clustering_n    | N/A                                 |
| clustering_u    | N/A                                 |

</td><td>

| |  |
|-----------------|-------------------------------------|
| Tree            | LG+I+G4                             |
| e-value_p       | 1e-160                              |
| e-value_n       | 1e-160                              |
| e-value_u       | 1e-160                              |
| coverage_p      | 98                                  |
| coverage_n      | 98                                  |
| coverage_u      | 98                                  |
| clustering_p    | N/A                                 |
| clustering_n    | N/A                                 |
| clustering_u    | N/A                                 |

</td></tr>
</table>


Redundant results for the prevalence, NCBI, and UBA queries were filtered from these BLASTP results by retrieving only the longest sequence for each uniquely labeled result.

##### Phylogenetic analysis

To compare the phylogenetic relationship of only the putative sequences, the sequences from the CARD prevalence data were concatenated to one multi-FASTA format file with the canonical sequences.
These the concatenated amino acid sequences were aligned with MAFFT-LINSI.
The aligned sequences were trimmed by trimal using the automated1 option.
IQ-TREE was then used to build a bootstrapped tree with -bb 1000 with the G+I+G4 model of substitution.

For the overall comparison of sequences, the filtered sequences from NCBI non-redundant data, CARD prevalence data, UBA data were all concatenated to one multi-FASTA format file with the canonical sequences.
These the concatenated amino acid sequences were aligned with MAFFT-LINSI.
The aligned sequences were trimmed by trimal using the automated1 option.
IQ-TREE was then used to build a bootstrapped tree with -bb 1000 with the G+I+G4 model of substitution.
All leaves of the trees were labeled with as much taxonomic information as possible for each rank, based on information from metadata [TODO: supplemental].
