## Methods

### Data

To sample diverse sources for AMR, various sequence databases were sampled for phylogenetic analyses.
These sources include CARD canonical and prevalence sequences for the AMR genes under study, and associated homologs from NCBI non-redundant data and metagenomic data.
The 32 canonical MCR sequence variants (table {@tbl:mcr-variants}), 14 canonical NDM sequence variants (table {@tbl:ndm-variants}), 6 canonical OXA-48-like sequence variants(table {@tbl:oxa-variants}), and 18 canonical KPC sequence variants (table {@tbl:kpc-variants}) as labeled in the CARD database, were obtained as a reference. The AMR prevalence data was queried from CARD Prevalence 3.0.5. 
The NCBI non-redundant data was queried from all non-redundant GenBank, PDB, SwissProt, PDB, PIR, and PRF on May 17, 2019.
The metagenomic data came from a data-set of 7903 draft quality MAGs which were recovered from the Sequence Read Archive by Parks @doi:10/cczd. These genomes were chosen specifically because they were likely to be from lineages which were under-sampled, environmental and non-human gastrointestinal samples being the main focus.

In the case of OXA beta-lactamase, only OXA-48-like genes were used for analysis. The OXA family is characterized by phenotype rather than genotype, and results in a low amount of sequence homology within the family. 
The phenotype of OXA-48 results from carbapenem hydrolyzing activity @doi:10/cn235v.
This subfamily of OXA contains homologous sequences suitable for this study.
Incorporating other subfamilies of OXA proved to be too cumbersome.


### Querying the data

The CARD canonical sequences from each family were used to perform a multiple query BLASTP (version 2.5.0 @doi:10/cnsjsz) against the prevalence BLAST database with a e-value threshold and query coverages shown in Table {@tbl:exp-params}. 
Many of the sequences are nearly identical, thus they were further processed by clustering with CD-HIT version 4.8.1 at a minimum sequence identities as per table {@tbl:exp-params}

The reference CARD canonical sequences were also used to perform a multiple query BLASTP against the NCBI non-redundant database with a e-value threshold, and query coverage, also indicated in Table  {@tbl:exp-params}
For simplicity in identifying the taxonomic history of the non-redundant hits, MULTISPECIES sequences were removed from the analyses.
There are many highly sampled taxa and genes in the non-redundant database. 
To balance the distribution, and reduce the size of the non-redundant sequence set, CD-HIT version 4.8.1 was also used to cluster the data as per table {@tbl:exp-params}.

For the metagenomic data, RGI version 5.0 with CARD database version 3.02  @doi:10/f9wbjs was run on the contigs of the 7903 MAGS with the inclusion of loose, perfect, and strict hits.
Sequences possibly containing AMR gene prediction data for each gene family was produced by filtering RGI output based on the its association with the search strings for each determinant in Table {@tbl:exp-params}
The filtered data were translated to a blast database.
The CARD canonical sequences were used to perform a multiple query BLASTP against this UBA blast database with a e-value threshold and  query coverage in Table {@tbl:exp-params}.

For all sequence variants obtained from each data source, redundant results for the prevalence, NCBI, and UBA queries were filtered from these BLASTP results by retrieving only the longest sequence for each uniquely labeled result. 

<table>
<td>
||
|-|
Table: [TODO: Make table more readable/better labels etc] e-value, query coverage, and cluster percentage used for each AMR family experiment for the prevalence, ncbi non-redundant, and UBA databases. {#tbl:exp-params}
</td>
<tr><th>(A) MCR phosphoethanolamine transferase </th>
<th>(B) KPC beta-lactamase</th>
<th>(C) NDM beta-lactamase</th>
<th>(D) OXA beta-lactamase</th></tr>

<tr><td>


| | |
|-----------------|-------------------------------------|
| Tree            | LG+I+G4                             |
| e-value_p       | 1e-160                              |
| e-value_n       | 1e-160                              |
| e-value_u       | 1e-160                              |
| coverage_p      | 98                                  |
| coverage_n      | 98                                  |
| coverage_u      | 98                                  |
| clustering_p    | 100                                 |
| clustering_n    | 100                                 |
| clustering_u    | 100                                 |

</td><td>

| | |
|-----------------|-------------------------------------|
| Tree            | LG+I+G4                             |
| e-value_p       | 1e-100                              |
| e-value_n       | 1e-40                              |
| e-value_u       | 1e-10                              |
| coverage_p      | 99                                  |
| coverage_n      | 99                                  |
| coverage_u      | 60                                  |
| clustering_p    | 99                                 |
| clustering_n    | 70                                 |
| clustering_u    | 100                                 |

</td><td>

| | |
|-----------------|-------------------------------------|
| Tree            | LG+I+G4                             |
| e-value_p       | 1e-160                              |
| e-value_n       | 1e-160                              |
| e-value_u       | 1e-10                              |
| coverage_p      | 98                                  |
| coverage_n      | 98                                  |
| coverage_u      | 60                                  |
| clustering_p    | 100                                 |
| clustering_n    | 100                                 |
| clustering_u    | 100                                 |

</td><td>

| |  |
|-----------------|-------------------------------------|
| Tree            | LG+I+G4                             |
| e-value_p       | 1e-100                              |
| e-value_n       | 1e-100                              |
| e-value_u       | 1e-10                              |
| coverage_p      | 90                                  |
| coverage_n      | 99                                  |
| coverage_u      | 95                                  |
| clustering_p    | N/A                                 |
| clustering_n    | N/A                                 |
| clustering_u    | N/A                                 |

</td></tr>
</table>


### Sequence Alignmnet

Two alignments were created for each AMR determinant under study.
The first alignment was created to compare the phylogenetic relationship of only the putative sequences.
The second alignment was created for an overall comparison of sequences. 

The putative AMR alignment was made up of the sequences from the CARD prevalence data were concatenated in one multi-FASTA format file with the canonical sequences and an outgroup chosen for each AMR family as per table {@tbl:outgroups}. 
In the second, more diverse alignment, The filtered sequences from NCBI non-redundant data, CARD prevalence data, UBA data were all concatenated to one multi-FASTA format file with the canonical sequences with the same outgroup sequences (table {@tbl:outgroups}).

This set of concatenated amino acid sequences were aligned with MAFFT-LINSI version 7.40 and trimmed by trimal version 1.4.rev22 using the automated1 option.

### Creation of phylogenies

For each alignment under each AMR family under study, IQ-TREE multicore version 1.6.9 @doi:10/f3srtd was used to build a bootstrapped tree with -bb 1000 with the G+I+G4 model of substitution. 
Tree visualizations were created with ETE Toolkit version 3, and annotated with taxonomic information for each rank, and environmental and AMR data, based on information from the various metadata. (See supplemental).
