## Methods
TODO: update these methods
TODO: Parameter sweeping ??

### AMR Prediction

A dataset of 7903 draft quality MAGs which were recovered from the Sequence Read Archive by Parks @doi:10/cczd.
TODO: source of UBA sequence files
RGI version 5.0 with CARD database version 3.02  @doi:10/f9wbjs was run on the contigs of the 7903 MAGS with the inclusion of loose, perfect, and strict hits.

### Phylogenetic Analysis

#### MCR

##### Obtaining candidatea and reference MCR genes

24 canonical sequence variants as indicated by CARD for the MCR phosphoethanolamine transferase [AMR Gene Family], MCR 3.11, 6.1, 2.2, 3.6, 3.10, 3.5, 1.10, 1.9, 3.9, 3.8, 3.7, 3.2, 3.1, 1.6, 1, 2, 3, 1.2, 1.3, 1.4, 8, 4, 5, 7.1 were obtained from the CARD database.

Output containing only AMR gene prediction data for the MCR phosphoethanolamine transferase gene family was produced by filtering RGI output based on a 'MCR phosphoethanolamine' string search.
The filtered data were translated to a blast database.
The 24 CARD canonical sequences were used to perform a multiple query BLASTP against the UBA blast database with a e-value threshold of 1e-180 and a query coverage of 90%.
Redundant results were filtered from these BLASTP results by retrieving only the longest sequence for each uniquely labeled result. 

Additional putative MCR phosphoethanolamine transferase gene family sequences were accumulated by querying the CARD prevalence data version 3.0.4 ("based on sequence data acquired from NCBI on 28-Feb-2019, analyzed using RGI 4.2.2 (DIAMOND homolog detection) and CARD 3.0.1").
The prevalence data was translated to a BLAST database.
The 24 CARD canonical sequences were used to perform a multiple query BLASTP against the prevalence blast database with a e-value threshold of 1e-180 and a query coverage of 90%.
Redundant results were filtered from these BLAST results by retrieving only the longest sequence for each uniquely labeled result.

The 24 CARD canonical sequences were used to perform a multiple query BLASTP against the NCBI non-redundant database blast database with a e-value threshold of 1e-180 and a query coverage of 90%.
Redundant results were filtered from these BLAST results by retrieving only the longest sequence for each uniquely labeled result.

The filtered sequences from NCBI non-redundant data, CARD prevalence data, UBA data were all concatenated to one multi-FASTA format file with the canonical sequences.
These the concatenated amino acid sequences were aligned with MAFFT FFT-NS-2.
TODO: re-run MAFFT with best parameters and update this file.
The aligned sequences were trimmed by trimal using the automated1 option.
IQ-TREE was then used to build a bootstrapped tree with -bb 1000 with the LG+R10 model of substitution.
all leaves of the trees were labeled with as much taxonomic information as possible for each rank, based on information from metadata.

The result was carried to subsequent analysis.
