## Methods

### AMR Prediction

### Materials

A data-set of 7903 draft quality MAGs which were recovered from the Sequence Read Archive by Parks @doi:10/cczd were used for this analysis.
RGI version 5.0 with CARD database version 3.02  @doi:10/f9wbjs was run on the contigs of the 7903 MAGS with the inclusion of loose, perfect, and strict hits.

CARD prevalence data version 3.0.4 ("based on sequence data acquired from NCBI on 28-Feb-2019, analyzed using RGI 4.2.2 (DIAMOND homolog detection) and CARD 3.0.1") was used to query the UBA data.
[TODO: Other versions of software]

#### MCR

##### Obtaining reference MCR genes

24 canonical sequence variants (see Table S1) as indicated by CARD for the MCR phosphoethanolamine transferase [AMR Gene Family], were obtained from the CARD database.

Additional putative MCR phosphoethanolamine transferase gene family sequences were accumulated by querying the CARD prevalence data.
The prevalence data was translated to a BLAST database.
The 24 CARD canonical sequences were used to perform a multiple query BLASTP against the prevalence blast database with a e-value threshold of 1e-180 and a query coverage of 99%.
Redundant results were filtered from these BLAST results by retrieving only the longest sequence for each uniquely labeled result.
Many of the sequences are nearly identical, thus they were further processed by clustering with CD-HIT at a minimum sequence identity of 99%

The 24 CARD canonical sequences were used to perform a multiple query BLASTP against the NCBI non-redundant database blast database with a e-value threshold of 1e-180 and a query coverage of 99%.
Redundant results were filtered from these BLAST results by retrieving only the longest sequence for each uniquely labeled result.
There are many highly sampled taxa and genes in the non-redundant database. To balance the distribution, and reduce the size of the non-redundant sequence set, CD-HIT was used to cluster the data at 70%.
 

##### Obtaining candidate MCR genes from the UBA data

Sequences possibly containing AMR gene prediction data for the MCR phosphoethanolamine transferase gene family was produced by filtering RGI output based on the its association with 'MCR phosphoethanolamine'.
The filtered data were translated to a blast database.
The 24 CARD canonical sequences were used to perform a multiple query BLASTP against this UBA blast database with a e-value threshold of 1e-170 and a query coverage of 99%.
For simplicity in identifying the taxonomic history of the non-redundant hits, MULTISPECIES sequences were removed from the analyses.
Redundant results were filtered from these BLASTP results by retrieving only the longest sequence for each uniquely labeled result.

##### Phylogenetic analysis of the MCR genes

To compare the phylogenetic relationship of only the putative sequences, the sequences from the CARD prevalence data were concatenated to one multi-FASTA format file with the canonical sequences.
These the concatenated amino acid sequences were aligned with MAFFT-LINSI.
The aligned sequences were trimmed by trimal using the automated1 option.
IQ-TREE was then used to build a bootstrapped tree with -bb 1000 with the G+I+G4 model of substitution.

For the overall comparison of sequences, the filtered sequences from NCBI non-redundant data, CARD prevalence data, UBA data were all concatenated to one multi-FASTA format file with the canonical sequences.
These the concatenated amino acid sequences were aligned with MAFFT-LINSI.
The aligned sequences were trimmed by trimal using the automated1 option.
IQ-TREE was then used to build a bootstrapped tree with -bb 1000 with the G+I+G4 model of substitution.

#### NDM

##### Obtaining reference NDM genes

14 canonical sequence variants (see Table S1) as indicated by CARD for the NDM beta-lactamase [AMR Gene Family], were obtained from the CARD database.

Additional putative NDM beta-lactamase gene family sequences were accumulated by querying the CARD prevalence data.
The prevalence data was translated to a BLAST database.
The 14 CARD canonical sequences were used to perform a multiple query BLASTP against the prevalence blast database with a e-value threshold of 1e-180 and a query coverage of 99%.
Redundant results were filtered from these BLAST results by retrieving only the longest sequence for each uniquely labeled result.
Many of the sequences are nearly identical, thus they were further processed by clustering with CD-HIT at a minimum sequence identity of 99%

The 14 CARD canonical sequences were used to perform a multiple query BLASTP against the NCBI non-redundant database blast database with a e-value threshold of 1e-180 and a query coverage of 99%.
Redundant results were filtered from these BLAST results by retrieving only the longest sequence for each uniquely labeled result.
CD-HIT was used to cluster the data at 70%.

##### Obtaining candidate NDM genes from the UBA data

Sequences possibly containing AMR gene prediction data for the NDM beta-lactamase gene family was produced by filtering RGI output based on the its association with 'NDM beta-lactamase'.
The filtered data were translated to a blast database.
The 14 CARD canonical sequences were used to perform a multiple query BLASTP against this UBA blast database with a e-value threshold of 1e-10 and a query coverage of 60%.
These values were loosened more for this analysis because at higher thresholds, there were no hits for UBA sequences in the results.
For simplicity in identifying the taxonomic history of the non-redundant hits, MULTISPECIES sequences were removed from the analyses.
Redundant results were filtered from these BLASTP results by retrieving only the longest sequence for each uniquely labeled result.

##### Phylogenetic analysis of the NDM genes

To compare the phylogenetic relationship of only the putative sequences, the sequences from the CARD prevalence data were concatenated to one multi-FASTA format file with the canonical sequences.
These the concatenated amino acid sequences were aligned with MAFFT-LINSI.
The aligned sequences were trimmed by trimal using the automated1 option.
IQ-TREE was then used to build a bootstrapped tree with -bb 1000 with the LG+I+G4 model of substitution.

For the overall comparison of sequences, the filtered sequences from NCBI non-redundant data, CARD prevalence data, UBA data were all concatenated to one multi-FASTA format file with the canonical sequences.
These the concatenated amino acid sequences were aligned with MAFFT-LINSI.
The aligned sequences were trimmed by trimal using the automated1 option.
IQ-TREE was then used to build a bootstrapped tree with -bb 1000 with the LG+I+G4 model of substitution.

#### OXA-48

##### Obtaining reference OXA-48-like genes

Because of the genetic diversity of the OXA gene family, only the OXA-48 like genes were used for analysis.
6 canonical sequence variants doi:10/db7w (see Table S1) OXA beta-lactamase [AMR Gene Family], were obtained from the CARD database.

Additional putative OXA beta-lactamase gene family sequences were accumulated by querying the CARD prevalence data.
The prevalence data was translated to a BLAST database.
The 6 CARD canonical sequences were used to perform a multiple query BLASTP against the prevalence blast database with a e-value threshold of 1e-100 and a query coverage of 99%.
Redundant results were filtered from these BLAST results by retrieving only the longest sequence for each uniquely labeled result.
Many of the sequences are nearly identical, thus they were further processed by clustering with CD-HIT at a minimum sequence identity of 99%

The 6 CARD canonical sequences were used to perform a multiple query BLASTP against the NCBI non-redundant database blast database with a e-value threshold of 1e-100 and a query coverage of 99%.
Redundant results were filtered from these BLAST results by retrieving only the longest sequence for each uniquely labeled result.
CD-HIT was used to cluster the data at 70%.

##### Obtaining candidate OXA-48-like genes from the UBA data

Sequences possibly containing AMR gene prediction data for the OXA beta-lactamase gene family was produced by filtering RGI output based on the its association with 'OXA beta-lactamase'.
The filtered data were translated to a blast database.
The 6 CARD canonical sequences were used to perform a multiple query BLASTP against this UBA blast database with a e-value threshold of 1e-10 and a query coverage of 95%.
For simplicity in identifying the taxonomic history of the non-redundant hits, MULTISPECIES sequences were removed from the analyses.
Redundant results were filtered from these BLASTP results by retrieving only the longest sequence for each uniquely labeled result.

##### Phylogenetic analysis of OXA-48-like genes

To compare the phylogenetic relationship of only the putative sequences, the sequences from the CARD prevalence data were concatenated to one multi-FASTA format file with the canonical sequences.
These the concatenated amino acid sequences were aligned with MAFFT-LINSI.
The aligned sequences were trimmed by trimal using the automated1 option.
IQ-TREE was then used to build a bootstrapped tree with -bb 1000 with the LG+I+G4 model of substitution.

For the overall comparison of sequences, the filtered sequences from NCBI non-redundant data, CARD prevalence data, UBA data were all concatenated to one multi-FASTA format file with the canonical sequences.
These the concatenated amino acid sequences were aligned with MAFFT-LINSI.
The aligned sequences were trimmed by trimal using the automated1 option.
IQ-TREE was then used to build a bootstrapped tree with -bb 1000 with the LG+I+G4 model of substitution.

#### KPC

##### Obtaining reference KPC genes

18 canonical sequence variants (see Table S1) KPC beta-lactamase [AMR Gene Family], as indicated by the card database, were obtained.

Additional putative KPC beta-lactamase gene family sequences were accumulated by querying the CARD prevalence data.
The prevalence data was translated to a BLAST database.
The 6 CARD canonical sequences were used to perform a multiple query BLASTP against the prevalence blast database with a e-value threshold of 1e-100 and a query coverage of 99%.
Redundant results were filtered from these BLAST results by retrieving only the longest sequence for each uniquely labeled result.
Many of the sequences are nearly identical, thus they were further processed by clustering with CD-HIT at a minimum sequence identity of 99%

The 6 CARD canonical sequences were used to perform a multiple query BLASTP against the NCBI non-redundant database blast database with a e-value threshold of 1e-40 and a query coverage of 99%.
Redundant results were filtered from these BLAST results by retrieving only the longest sequence for each uniquely labeled result.
CD-HIT was used to cluster the data at 70%.

##### Obtaining candidate KPC genes from the UBA data

Sequences possibly containing AMR gene prediction data for the KPC beta-lactamase gene family was produced by filtering RGI output based on the its association with 'KPC beta-lactamase'.
The filtered data were translated to a blast database.
The 6 CARD canonical sequences were used to perform a multiple query BLASTP against this UBA blast database with a e-value threshold of 1e-10 and a query coverage of 60%.
For simplicity in identifying the taxonomic history of the non-redundant hits, MULTISPECIES sequences were removed from the analyses.
Redundant results were filtered from these BLASTP results by retrieving only the longest sequence for each uniquely labeled result.

##### Phylogenetic analysis of KPC genes

To compare the phylogenetic relationship of only the putative sequences, the sequences from the CARD prevalence data were concatenated to one multi-FASTA format file with the canonical sequences.
These the concatenated amino acid sequences were aligned with MAFFT-LINSI.
The aligned sequences were trimmed by trimal using the automated1 option.
IQ-TREE was then used to build a bootstrapped tree with -bb 1000 with the LG+I+G4 model of substitution.

For the overall comparison of sequences, the filtered sequences from NCBI non-redundant data, CARD prevalence data, UBA data were all concatenated to one multi-FASTA format file with the canonical sequences.
These the concatenated amino acid sequences were aligned with MAFFT-LINSI.
The aligned sequences were trimmed by trimal using the automated1 option.
IQ-TREE was then used to build a bootstrapped tree with -bb 1000 with the LG+I+G4 model of substitution.

##### Labeling of phylogenies with metadata

All leaves of the trees were labeled with as much taxonomic information as possible for each rank, based on information from metadata as per table S2.
