## Methods

### Data

To sample diverse sources for AMR, various sequence databases were sampled for phylogenetic analyses.
These sources include CARD canonical and prevalence sequences for the AMR genes under study, and associated homologs from NCBI non-redundant data and metagenomic data.
The 32 canonical MCR sequence variants (table {@tbl:mcr-variants}), 14 canonical NDM sequence variants (table {@tbl:ndm-variants}), 6 canonical OXA-48-like sequence variants(table {@tbl:oxa-variants}), and 18 canonical KPC sequence variants (table {@tbl:kpc-variants}) as labeled in the CARD database, were obtained as a reference. The AMR prevalence data was queried from CARD Prevalence 3.0.5. 
The NCBI non-redundant data was queried from all non-redundant GenBank, PDB, SwissProt, PDB, PIR, and PRF on May 17, 2019.
The metagenomic data came from a data-set of 7903 draft quality MAGs which were recovered from the Sequence Read Archive by Parks @doi:10/cczd. These genomes were chosen specifically because they were likely to be from lineages which were under-sampled, environmental and non-human gastrointestinal samples being the main focus.

In the case of OXA beta-lactamase, only OXA-48-like genes were used for analysis. The OXA family is characterized by phenotype rather than genotype, and results in a low amount of sequence homology within the family. 
The phenotype of OXA-48 results from carbapenem hydrolyzing activity @doi:10/cn235v.
This subfamily of OXA contains homologous sequences suitable for this study.
Incorporating other subfamilies of OXA proved to be too cumbersome.


### Querying the data

The CARD canonical sequences from each family were used to perform a multiple query BLASTP (version 2.5.0 @doi:10/cnsjsz) against the prevalence BLAST database with a e-value threshold and query coverages shown in Table 1. 
Many of the sequences are nearly identical, thus they were further processed by clustering with CD-HIT version 4.8.1 at a minimum sequence identities as per table {@tbl:exp-params}

The reference CARD canonical sequences were also used to perform a multiple query BLASTP against the NCBI non-redundant database with a e-value threshold, and query coverage, also indicated in Table 1
There are many highly sampled taxa and genes in the non-redundant database. 
To balance the distribution, and reduce the size of the non-redundant sequence set, CD-HIT version 4.8.1 was also used to cluster the data as per table {@tbl:exp-params}.

For the metagenomic data, RGI version 5.0 with CARD database version 3.02  @doi:10/f9wbjs was run on the contigs of the 7903 MAGS with the inclusion of loose, perfect, and strict hits.
Sequences possibly containing AMR gene prediction data for each gene family was produced by filtering RGI output based on the its association with the search strings for each determinant in Table 1
The filtered data were translated to a blast database.
The CARD canonical sequences were used to perform a multiple query BLASTP against this UBA blast database with a e-value threshold and  query coverage in Table 1. The E-values were chosen such that a tractable or sufficient
number of sequences resulted from their respective database query. The minimum BLAST query coverages were chosen as to reduce noise in the final alignment, thus reducing the possibility of attempting to align non-homologous sequences.

For all sequence variants obtained from each data source, redundant results for the prevalence, NCBI, and UBA queries were filtered from these BLASTP results by retrieving only the longest sequence for each uniquely labeled result. 

<style type="text/css">
.tg  {border-collapse:collapse;border-spacing:0;}
.tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
  overflow:hidden;padding:10px 5px;word-break:normal;}
.tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
  font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
.tg .tg-yj5y{background-color:#efefef;border-color:inherit;text-align:center;vertical-align:top}
.tg .tg-j4pq{background-color:#efefef;border-color:#000000;text-align:center;vertical-align:top}
.tg .tg-llyw{background-color:#c0c0c0;border-color:inherit;text-align:left;vertical-align:top}
.tg .tg-0pky{border-color:inherit;text-align:left;vertical-align:top}
.tg .tg-0lax{text-align:left;vertical-align:top}
</style>
<table class="tg">
<thead>
  <tr>
    <th class="tg-yj5y" rowspan="2">AMR<br>Determinant</th>
    <th class="tg-yj5y" colspan="3">Minimum BLAST<br>E-value<br></th>
    <th class="tg-yj5y" colspan="3">Minimum BLAST <br>Query Coverage</th>
    <th class="tg-j4pq" colspan="3">CD-HIT Sequence <br>Identity Threshold</th>
  </tr>
  <tr>
    <td class="tg-llyw">CARD<br></td>
    <td class="tg-llyw">NCBI<br></td>
    <td class="tg-llyw">Parks<br></td>
    <td class="tg-llyw">CARD<br></td>
    <td class="tg-llyw">NCBI<br></td>
    <td class="tg-llyw">Parks<br></td>
    <td class="tg-llyw">CARD<br></td>
    <td class="tg-llyw">NCBI<br></td>
    <td class="tg-llyw">Parks<br></td>
  </tr>
</thead>
<tbody>
  <tr>
    <td class="tg-0pky">MCR<br></td>
    <td class="tg-0pky">1e-160</td>
    <td class="tg-0pky">1e-160</td>
    <td class="tg-0pky">1e-160</td>
    <td class="tg-0pky">98</td>
    <td class="tg-0pky">98</td>
    <td class="tg-0pky">98</td>
    <td class="tg-0pky">100</td>
    <td class="tg-0pky">100</td>
    <td class="tg-0pky">100</td>
  </tr>
  <tr>
    <td class="tg-0pky">KPC</td>
    <td class="tg-0pky">1e-100</td>
    <td class="tg-0pky">1e-40</td>
    <td class="tg-0pky">1e-10</td>
    <td class="tg-0pky">99</td>
    <td class="tg-0pky">99</td>
    <td class="tg-0pky">60</td>
    <td class="tg-0pky">99</td>
    <td class="tg-0pky">70</td>
    <td class="tg-0pky">100</td>
  </tr>
  <tr>
    <td class="tg-0pky">NDM</td>
    <td class="tg-0pky">1e-160</td>
    <td class="tg-0pky">1e-160</td>
    <td class="tg-0pky">1e-10</td>
    <td class="tg-0pky">98</td>
    <td class="tg-0pky">98</td>
    <td class="tg-0pky">60</td>
    <td class="tg-0pky">100</td>
    <td class="tg-0pky">100 </td>
    <td class="tg-0pky">100</td>
  </tr>
  <tr>
    <td class="tg-0pky">OXA</td>
    <td class="tg-0pky">1e-100</td>
    <td class="tg-0pky">1e-100</td>
    <td class="tg-0pky">1e-10</td>
    <td class="tg-0pky">90</td>
    <td class="tg-0pky">99</td>
    <td class="tg-0pky">95</td>
    <td class="tg-0pky">N/A</td>
    <td class="tg-0pky">N/A</td>
    <td class="tg-0pky">N/A</td>
  </tr>
  <tr>
    <td class="tg-0lax" colspan="10"><b>Table 1</b>: Parameters used with BLAST and CD-HIT to obtain sequences for the phylogenies<br>of MCR phosphoethanolamine transferase, and KPC, NDM, and OXA beta-lactamases.<br>For each determinant, the minimum BLAST E-values, minimum query coverages, and CD-HIT Sequence Identity <br>threshold for clustering sequences are displayed for each database.<br>CARD represents the RGI-CARD prevalence sequence data, NCBI represents the NCBI non-redundant sequence<br>data, and Parks represents the MAG data source.<br></td>
  </tr>
</tbody>
</table>


### Sequence Alignment

Two alignments were created for each AMR determinant under study.
The first alignment was created to compare the phylogenetic relationship of only the putative sequences.
The second alignment was created for an overall comparison of sequences. 

The putative AMR alignment was made up of the sequences from the CARD prevalence data were concatenated in one multi-FASTA format file with the canonical sequences and an outgroup chosen for each AMR family as per table {@tbl:outgroups}. 
In the second, more diverse alignment, The filtered sequences from NCBI non-redundant data, CARD prevalence data, UBA data were all concatenated to one multi-FASTA format file with the canonical sequences with the same outgroup sequences (table {@tbl:outgroups}).

This set of concatenated amino acid sequences were aligned with MAFFT-LINSI version 7.40 and trimmed by trimal version 1.4.rev22 using the automated1 option.

### Creation of phylogenies

For each alignment under each AMR family under study, IQ-TREE multicore version 1.6.9 @doi:10/f3srtd was used to build a bootstrapped tree with -bb 1000 with the G+I+G4 model of substitution. 
Tree visualizations were created with ETE Toolkit version 3.
The trees were annotated with taxonomic information for each rank by using ETEâ€™s NCBI taxonomy module to translate the names result set from the various databases to taxonomic identifiers.
The taxonomic identifier was used to query NCBI's taxonomy database for the taxonomy information.
In cases where there were multiple hits for genes, the most recent common ancestor was used to query the taxonomic information, therefore some leaves in the trees were annotated with only partial taxonomic information.

