## Results

### Phylogenetic analysis of putative MCR sequences

The phylogenetic relationships of the CARD canonical sequences, and the CARD prevalence sequences involving the MCR family were investigated.
A total of 30 genes, an out-group, the 23 canonical sequences, and the 6 prevalence sequences, were selected for analyses.
The tree in Figure {@fig:canon-prev-tree} shows several distinct clades.
Each MCR variant forms its own clade.
MCR-1, MCR-2, and MCR-6 form a clade, appearing to more closely related to one another than with the other MCR family members.

![Phylogenetic relationship for 23 canonical (lcl_canon) MCR family sequences along with 6 prevalence (lcl_prev) sequences.](images/mcr190905_canon_prev.fasta.collapsed.svg){#fig:canon-prev-tree}


### Phylogenetic analysis of NCBI data

Sequences from the NCBI non-redundant data were added to the analysis to be compared with the canonical and prevalence sequences.
This resulted in a total of 409 sequences for subsequent analysis, all labeled as phosphoethanolamine lipid A transferase genes, where 104 hits were labeled as MCR family genes.

One of the BLAST results was a variant which was not included in the CARD canonical or prevalence sequences.
The variant, MCR-9, was reported in the literature to be a novel MCR homologue with similarity to MCR-3 @doi:10/gf7r4q. The gene was shown to confer resistance to colistin when expressed in *Escherichia coli*.

The phylogeny in Figure {@fig:canon-prev-nrdb-uba-tree} shows that the, the MCR 3 clade branches with MCR-9.
This MCR-3 and MCR-9 clade is separated by a large amount diversity from the MCR 1,2,4,5, and 8 clades.
This diversity is represented by many bacteria containing phosphoethanolamine lipid A transferase with none reported to be in the MCR family.

![Phylogenetic relationship (lcl_canon) MCR family sequences along with 6 prevalence (lcl_prev) sequences.](images/mcr190905_canon_prev_nrdb_uba.fasta.svg){#fig:canon-prev-nrdb-uba-tree}

### Phylogenetic analysis of UBA sequences with RGI-CARD

RGI produced 1457246 results AMR determinants under the loose cutoff from the UBA data, 7171 for the strict cutoff,and 310 for the loose cutoff.
It was hoped that pyhlogenetic analysis could find AMR determinants would be found in under the loose criterion that may have been missed by RGI-CARD.
The UBA BLAST results were included in the phylogeny in Figure {@fig:canon-prev-nrdb-uba-tree} for the analysis.
The diversity between clades, swathes of which are not reported to be in the MCR family, contain multiple sources from UBAs.
It is unclear if these relationships could be involved with the MCR family, but there are more enticing relationships present in the tree.
The remainder of the analysis deals with relationships which are deemed to be interesting based on the locations of the UBAs between MCR family clades.


#### MCR-5, MCR-1,2,6 clades

The MCR-5, MCR1,2,6clades shown in figure ?? share, with 100 bootstrap support, a common ancestor.
Branching between these two clades, as a descendant to this ancestor, is a clade of Psycrobacter species bacteria.
This clade includes two hits from the non-redundant database, and two hits from the UBA data.
According to the Parks @doi:10/cczd data, the identity of these samples are,
UBA3068, A Psychrobacter sp., sampled from Oil polluted marine microbial communities from Coal and Oil in Point Santa Barbara, California, USA
and, UBA4193, a Psychrobacter sp, sampled from the New York City MTA subway samples.
The quality information for these sequences, shown in Table ??, shows that UBA3068 is near complete, while UBA4193 is only partial.
It is encouraging to see that even with the quality difference, these two sequences branch in the expected clade.
Psychrobacter @doi:10/dbpr is a Genus is widespread and includes many cold adapted bacteria, it is an opportunistic pathogen, and has been found sometimes be a cause of infections in humans, animals, and fish. Many new species of Psychrobacter have been discovered in cold climates @doi:10/fd9rb9. Some of the species have been shown to be resistant to colistin, like Psychrobacter vallis ps. nov. and Psychrobacter aquaticus ps. nov. @doi:10/cdq9w4.


#### MCR-3 clade
A single result, UBA705, which the Parks data @doi:10/cczd reports as a Comparative metagenome analyses of anode-associated microbial communities developed in rice paddy field-soil microbial fuel cells, is reported to be Aeromonas hydrophilia.
This present within the clade alone with several other canonical and non-redundant Aeromonadalacea.
Aeromonas hydrophilia is a species which has been found to have an MCR-3 gene.
Again, even though qualities vary (Table ??), the sequences branch in the expected location.

#### MCR 3,9 clade
This clade contains 8 UBAs from the loose CARD hits under the given constraints, and several NCBI non-redundant hits not reported as MCR family genes.
7 Loose hits for MCR in a /Leclercia/ adecarboxylata branch within this clade, branching closely with the NCBI non-redundant MCR9. 
These /Leclercia/ branch below the common ancestor of MCR-9 and MCR-3 which are well supported. The /Leclercia/ UBAs were all sampled from New York City MTA subway samples Metagenome.
Acinetobacter is another opportunistic pathogen  @doi:10/b2zr9k which is becoming resistant to many antimicrobials.