# Supplemental


|MCR-9 MCR-1.8 MCR-1.11 MCR-3.4 MCR-3.11 MCR-6.1 MCR-3.3 MCR-2.2 MCR-1.5 MCR-3.6 MCR-3.10 MCR-3.5 MCR-1.10 MCR-1.9 MCR-3.9 MCR-3.8 MCR-3.7 MCR-3.2 MCR-1.6 MCR-1.2 MCR-1.3 MCR-1.4 MCR-8 MCR-7.1 MCR-1.7 MCR-1.12 MCR-1.13 MCR-3.12 MCR-4.2 MCR-4.3 MCR-4.4 MCR-4.5 MCR-5.2|
|-|
Table: Names of the canonical MCR phosphoethanolamine transferase variants from CARD database {#tbl:mcr-variants}


|KPC-1 KPC-3 KPC-4 KPC-5 KPC-6 KPC-7 KPC-8 KPC-9 KPC-10 KPC-11 KPC-12 KPC-13 KPC-14 KPC-15 KPC-16 KPC-17 KPC-19 KPC-22 KPC-24|
|-|
Table: Names of the canonical beta-lactamase variants from CARD database {#tbl:kpc-variants}


|NDM-1 NDM-2 NDM-5 NDM-3 NDM-4 NDM-6 NDM-7 NDM-8 NDM-9 NDM-10 NDM-11 NDM-12 NDM-13 NDM-14 NDM-17|
|-|
Table: Names of the canonical NDM beta-lactamase variants from CARD database {#tbl:ndm-variants}


|OXA-162 OXA-163 OXA-181 OXA-204 OXA-232 OXA-48|
|-|
Table: Names of the canonical OXA beta-lactamase variants from CARD database {#tbl:oxa-variants}

TODO: Quality/CHECKM information for UBAs

| Outgroup to AMR Family | Gene                                         | Species                | Class                |
|------------------------|----------------------------------------------|------------------------|----------------------|
| MCR                    | phosphoethanolamine--lipid A transferase     | Oxalobacter formigenes | Betaproteobacteria   |
| KPC                    | class A beta-lactamase                       | Mesorhizobium loti     | Alpha Proteobacteria |
| NDM                    | VIM-4 metallo-beta-lactamase                 | Alcaligenes faecalis   | Betaproteobacteria   |
| OXA-48                 | OXA-10 family class D beta-lactamase OXA-454 | Delftia acidovorans    | Betaproteobacteria   |
Table: Outgroups chosen in building the phylogenies of the 4 AMR families. {#tbl:outgroups}


#### MCR Alignment
[TODO: visualization with alignment/entropy]
![placeholder](images/aln_mcr.png){#fig:ndm-tree}

#### KPC Alignment
[TODO: visualization with alignment/entropy]
![placeholder](images/aln_kpc.png){#fig:ndm-tree}

#### NDM Alignment
[TODO: visualization with alignment/entropy]
![placeholder](images/aln_ndm.png){#fig:ndm-tree}

#### OXA-48 Alignment
[TODO: visualization with alignment/entropy]
![placeholder](images/aln_oxa.png){#fig:ndm-tree}


![Phylogenetic relationship of 32 canonical (labels prefixed with lcl_canon_ in yellow), and 54 prevalence (labels prefixed with lcl_prev_ in tan) MCR family sequences, 595 NCBI non-redundant sequences (labels prefixed with lcl_prev_ in blue), and 91 UBA sourced sequences (labels prefixed with lcl_prev_ in green). Each MCR variant is coloured based on its primary numerical value. If the sequence is not reported to be MCR family it is coloured in grey.](images/mcr191004_canon_prev_nrdb_uba.fasta.aln.trim.treefile.expanded.svg){#fig:canon-prev-nrdb-uba-tree}

![Phylogenetic relationship (lcl_canon) OXA-48 family sequences along with 14  prevalence (lcl_prev) sequences.](images/oxa191128.canon.svg){#fig:oxa-tree}

![Phylogenetic relationship (lcl_canon) OXA-48 family sequences along with 14  prevalence (lcl_prev) sequences.](images/oxa191128.expanded.svg){#fig:oxa-tree-all}

![Phylogenetic relationship (lcl_canon) NDM family sequences along with 14  prevalence (lcl_prev) sequences.](images/ndm191128.canon.svg){#fig:ndm-tree}

![Phylogenetic relationship (lcl_canon) KPC family sequences along with 14  prevalence (lcl_prev) sequences.](images/kpc191127.canon.svg){#fig:kpc-tree}


![Phylogenetic relationship (lcl_canon) KPC family sequences along with 14  prevalence (lcl_prev) sequences.](images/kpc191127_canon_prev_nrdb_uba.fasta.aln.trim.treefile.expanded.svg){#fig:kpc-tree-all}
